kimchi/src/linearization.rs

```diff
@@ -151,10 +186,11 @@ pub fn linearization_columns<F: FftField + SquareRootField>(
 ///
 /// Will panic if the `linearization` process fails.
 pub fn expr_linearization<F: PrimeField + SquareRootField>(
-    feature_flags: &FeatureFlags<F>,
+    feature_flags: Option<&FeatureFlags<F>>,
```

Comment:
I'm not sure I understand why this is better. Now the code above is quite more complex. What's the rationale behind passing a `None` being better instead of a `FeatureFlags::default()`?

Reply:
ok I think I'm starting to understand, you're choosing to decide how the expression looks like at compile time (Some) or at runtime (None)

Reply:
maybe good to add some comment somewhere to say exactly this ^, this would help users of this function